src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo_Source.cs

```diff
@@ -54,7 +54,7 @@ private static void FreeSymbolMap(MultiDictionary<string, ISymbol> symbolMap)
 
             // Order the documents by FilePath.  Default ordering in the RemoteWorkspace is
```

Comment:
Agreed, as @jasonmalinowski said, order matters for correctness to the compiler, so not having the same order might actually be a source of bugs.